loader/core-plugins.js

```diff
@@ -0,0 +1,79 @@
+const CordlrPlugin = require('cordlr-plugin')
+
+class HelpPlugin extends CordlrPlugin {
```

Comment:
The `core-plugins should actually be in the folder cordlr-cli/loader/plugins/help.js` because it's easier to just organize multiple plugins that way.